requirements.txt

```diff
@@ -1,3 +1,3 @@
 psutil>=3.2.2
-signalfx>=1.0.1
+signalfx>=1.0.4
```

Comment:
I don't think we should do == because then they can't install newer versions of the python api and use our plugin.  I think this is fine as if they upgrade to this version of the plugin they'll upgrade the python and no breakage will occur, same for fresh install.